qstnrs/questionnaire/migrations/0003_auto__add_result__chg_field_questionnaire_result_upper_limit__chg_fiel.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+        # Adding model 'Result'
+        db.create_table('questionnaire_result', (
```

Comment:
I don't think this is working. This is the second table with this name 'questionnaire_result'. You already created one in the first migration and never deleted it. I think you can drop this one.